src/backend/controllers/tankController.js

```diff
@@ -1,64 +1,67 @@
 // @flow strict
 
+// Flow type import
+import type { Request, Response } from 'express';
+
 const User = require('../../models/userModel');
 const Tank = require('../../models/tankModel');
 
 const { validationResult } = require('express-validator');
 
-// Flow type import
-import type {
-	$Request,
-	$Response,
-	NextFunction,
-	Middleware,
-} from 'express';
-
-exports.getFavorite = async (req: $Request, res: $Response) => {
+exports.getFavorite = async (req: Request, res: Response) => {
 	await User.findById(req.user.id, 'favoriteTankId', (err: Error, myUser: User) => {
 		if(err) {
 			res.send(err);
 			console.log('could not find user');
 		} 
 		else
```

Comment:
err, I think you are missing curly braces on your else if you want the retrieved... to only be in the else?